.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
     "editor.codeActionsOnSave": {
         "source.fixAll.eslint": true
     },
-    "eslint.validate": ["javascript"]
+    "eslint.validate": ["javascript"],
+    "editor.formatOnSave": false
```

Comment:
particular reason?

Reply:
The prettier formatting was conflicting with the eslint config. So I turned it off when I'm working on it. Will revert it again. Also, I changed some formatting rules from eslint because of https://prettier.io/docs/en/comparison.html#how-does-it-compare-to-eslinttslintstylelint-etc reason and can you please update the eslint config to remove all the formatting rules please,

Reply:
kk